src/tests/ConnectionManagerTest.java

```diff
@@ -0,0 +1,21 @@
+package tests;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import database.ConnectionManager;
+
+public class ConnectionManagerTest {
```

Comment:
This class is not required here, the class on source `test` is already accomplishing this class purpose.